pkg/api/customization/logging/validator.go

```diff
@@ -75,8 +75,10 @@ func validate(level, containerLogSourceTag string, loggingTargets v3.LoggingTarg
 		}
 	}
 
-	if err = generator.ValidateCustomTags(wrap); err != nil {
-		return err
+	if len(outputTags) != 0 {
+		if err = generator.ValidateCustomTags(wrap); err != nil {
```

Comment:
I see that the same pattern is being repeated here and on line 51. Would it make sense to include len(outputTags) != 0 verification in ValidateCustomTags method itself, and just return from the method immediately if ==0?

Reply:
I agree this `!=0` is being repeated, I check the method ValidateCustomTags, we pass interface{} as method's parameters, because we may pass ProjectLoggingTemplateWrap or ClusterLoggingTemplateWrap and use them the generate the configure. So if we may need to parse it back to map if we move the `!=` into ValidateCustomTags method, I prefer to keep it outside even it is being repeated

Reply:
ok, it's fine, not a big deal